ibraheemosule/skoolizy
# updated settings page with multiple sections

# Why this PR?
To add functionality on the settings page

# What does this PR do?
It adds to key sections to the settings page
- Notification Settings that includes toggles for the notification preference , A reusable toggle component was created for this purpose.
- Academic Term Settings that features a date picker that allows users define the start and end dates for academic terms

# How to test your work?
-Navigate to the Settings page.
-Check for the Notifications section, where you should see toggle switches for different notification options.
-Look for the Academic Term Settings section, which should include a date picker for setting the start and end dates of the academic terms.
-Test the functionality of the toggles and the date picker to ensure they work as intended.

# Pages to see your changes?
/settings
